website/docs/cdktf/concepts/stacks.mdx

```diff
@@ -190,7 +190,7 @@ If you e.g. write `this.allResources = Fn.merge(resourceFromStackA.items, resour
 In case you don't want the "middleman" to be cut out you can run `stackD.addDependency(stackC)` to make the dependency explicit.
 If you want to keep the result of the function attached to one stack (and saved its state) you can use a `TerraformLocal` and expose it instead: `this.allResources = TerraformLocal(this, "merged_items", Fn.merge(resourceFromStackA.items, resourceFromStackB.items)`.
 
-There are currently no safe-guards around this feature, it's your responsibility to deploy the stacks in the respective order.
+The CLI will error if you try to deploy something without deploying the dependencies first and if you try to destroy something without destroying the dependant stacks first. If you don't want these safe-guards you can add the `--ignore-missing-stack-dependencies` to deploy / destroy command.
```

Comment:
Wording edits. Also - I"m confused by this. Doesn't terraform automatically make sure that dependencies are taken into account? Does that not apply here because these are CDKTF concepts that terraform doesn't know about?

```suggestion
The CLI will error if you deploy your application without first deploying the dependencies. It will also error if you try to destroy infrastructure without destroying the dependent stacks first. To remove these safeguards,  add the `--ignore-missing-stack-dependencies` to the `deploy` and `destroy` commands.
```

Reply:
> Does that not apply here because these are CDKTF concepts that terraform doesn't know about?

Yes, this is the equivalent of having two Terraform projects and deploying them one after another (which the Terraform CLI itself can't do as far as I know).

Reply:
ooo makes sense! Thank you :-)